examples/circom.rs

```diff
@@ -0,0 +1,169 @@
+//! # Circom Gadgets
+//!
+//! ## Setting up a Circom Gadget with Lurk
+//!
+//! First, in a separate directory from `lurk-rs`, clone the circomlib repo with the `Sha256_2` circuit.
+//! ```
+//! git clone git@github.com:iden3/circomlib.git && cd circomlib
+//! ```
+//! In the `circuits` folder, create a new file called `sha256_2.circom`.
+//! ```
+//! cd circuits && touch sha256.circom
+//! ```
+//! In this new file, create the main component that simply calls the `Sha256_2` circuit.
+//! ```
+//! pragma circom 2.1.6;
+//!
+//! include "circomlib/sha256/sha256_2.circom";
```

Comment:
I needed the following change:
```suggestion
//! include "./sha256/sha256_2.circom";
```